macros/models/_sector/transfers/erc20/transfers_erc20.sql

```diff
@@ -4,99 +4,103 @@ WITH
 
 
 erc20_transfers  as (
-        SELECT 
-            'receive' as transfer_type, 
+        SELECT
+            'receive' as transfer_type,
             evt_tx_hash,
-            evt_index, 
+            evt_index,
             evt_block_time,
-            to as wallet_address, 
+            to as wallet_address,
             contract_address as token_address,
-            CAST(value as double) as amount_raw
-        FROM 
+            try_cast(value as int256) as amount_raw
```

Comment:
It seems to me that this would produce some data loss on some potentially relevant pieces of data. Are we confident these are not relevant cases?

For example, a treasury address receiving more than half the token supply of a token which total supply is `2^256-1` would be filtered out by the `try_cast` here.

See my comment there: https://github.com/duneanalytics/spellbook/issues/4521#issuecomment-1754028324